src/StreamDeck.js

```diff
@@ -211,6 +218,11 @@ class StreamDeck { // eslint-disable-line
             device.productId === 0x0080) {
           return device;
         }
+      } else if (
```

Comment:
the WET-ness of this function stood out to me before (`getPreviousDevice` repeating a bunch from `getDevice`).. and this new branch exacerbates it a bit..

so I tried to DRY it up.. but tbh I don't *love* the result:  https://github.com/paulirish/StreamDeck-Meet/compare/macropad...dry-er-filters
I'm pretty confident the match-any-filter impl I have matches up completely with the spec, so it does seem very future-friendly. But it isn't nice to look at.

Your call. I could totally go either way.